src/config-loader/config-loader.coffee

```diff
@@ -2,6 +2,7 @@ _ = require 'lodash'
 Promise = require 'bluebird'
 sbvrUtils = require '../sbvr-api/sbvr-utils'
 permissions = require '../sbvr-api/permissions'
+resourceaccess = require '../sbvr-api/resource-access'
```

Comment:
I feel I'm commenting too many NITs but, can you upper case the `a` so it's proper camel case?

Reply:
Don't worry you are totally right, the upper case `A` makes it much easier to read. :)